app/src/test/report/reducers/ReportsReducersTests.ts

```diff
@@ -1,31 +0,0 @@
-import { expect } from "chai";
```

Comment:
Noticed that we had 2 files of report reducer tests. Have moved these 4 tests into the other file.